frontend/templates/modals/receipts_upload.html

```diff
@@ -46,6 +46,7 @@
         </label>
         <input required
                type="number"
```

Comment:
Does number need to be changed to something else for this to work?

Reply:
Nope, its still necessary for number to be there so only numbers are accepted. The step attribute just enables up to two decimals on that number